CONTRIBUTING.md

```diff
@@ -0,0 +1,110 @@
+# Contributing to the Project
+
+This document contains and defines the rules that have to be followed by any
+contributor to the project, in order for any change to be merged into the
+stable branches.
+
+## Workflow
+
+The workflow followed is waterfall. In short, a contributor will target his 
+work on the lowest version branch possible starting with 2.0, and the changes
+will automatically be forward-ported to higher versions.
+
+## Restrictions of the master branch
+
+The master branch is a very specific branch in our workflow. No commit must
+ever go directly into the master branch, and the code shall follow one and only
+one path to get into the master branch.
+
+Merging a pull request into the master branch will only ever be done by the
+Bert-E merging bot. Once the necessary approvals and CI passes are met, one 
+only needs to comment `@Bert-E_scality approve` to trigger the merging process
```

Comment:
Isn't it simply `@bert-e approve`?

Reply:
going on memory from bitbucket workflow doesn't work too well :upside_down_face: